src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,51 @@
 import React from 'react';
 
 import './MovieCard.scss';
+import { Movie } from '../../types/Movie';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+type Props = {
+  movie: Movie;
+};
+
+export const MovieCard: React.FC<Props> = ({ movie }) => (
```

Comment:
You can destruct movie and not write movie.imgUrl, movie.title and other everywhere, just title, imgUrl, but I think it is not so critical

Reply:
Yeah, use destructuring here from movie